frontend/src/utils/languages.jsx

```diff
@@ -0,0 +1,32 @@
+
+export const LANGUAGE_VERSIONS = {
+    javascript: "18.15.0",
+    typescript: "5.0.3",
+    python: "3.10.0",
+    java: "15.0.2",
+    csharp: "6.12.0",
+    php: "8.2.3",
+    cpp: "11.1",
+    c: "11.1",
+    ruby: "3.1",
+    go: "1.18",
+    swift: "5.5",
+    kotlin: "1.6",
+  };
```

Comment:
Should we hard code the versions? or is there a better way to do this

Reply:
if the api is only using the latest version we can delete this from here?